### ISSUE TITLE
link to talk proposal form

### ISSUE THREAD

**Issue by phette23** [code4lib/2017.code4lib.org#18]
need buttons (& maybe one sentence teaser?) on:
- home page
- schedule/timeline.html

**Comment by cbeer** [comment]
Can someone also be sure to update http://code4lib.org/conference/2016?

**Comment by phette23** [comment]
Do you mean http://2016.code4lib.org/? Or both?

Actually brings up a good issue, we don't have any maintenance/update plan for the older sites. We should figure out the most minimal, effective way to direct people over to the current year's conference from prior ones.

**Comment by cbeer** [comment]
oh, sorry, I should have read the content. Let me rephrase - you should have a 2017 Drupal page.

**Comment by phette23** [comment]
I've created a draft but how do I change the URL? Drupal fail. Seems like I'm either missing something or lack permissions.

http://code4lib.org/node/585/edit

**Comment by cbeer** [comment]
I'll take care of it, thanks.

**Comment by phette23** [comment]
awesome thanks! I think we can close this now.
